cmd/podman/libpodruntime/runtime.go

```diff
@@ -15,30 +15,35 @@ import (
 
 // GetRuntimeMigrate gets a libpod runtime that will perform a migration of existing containers
 func GetRuntimeMigrate(ctx context.Context, c *cliconfig.PodmanCommand, newRuntime string) (*libpod.Runtime, error) {
-	return getRuntime(ctx, c, false, true, false, true, newRuntime)
+	return getRuntime(ctx, c, false, true, false, false, true, newRuntime)
```

Comment:
I think it is time to make getRuntime take a structure, real tough to read this and understand what is going on, for future users.

Reply:
I do this in https://github.com/containers/libpod/pull/5126 and will rebase later on top here if merged.